examples/verrazzano/README.md

```diff
@@ -0,0 +1,51 @@
+# Installing Verrazzano on OKE

```

Comment:
Need to spell out Oracle Container Engine for Kubernetes (OKE) on the first use in each file.